dotcom-rendering/src/web/components/PinnedPostLiveness.importable.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from 'react';
+
+const isServer = typeof window === 'undefined';
+
+const pinnedPost: Element | null = !isServer
+	? window.document.querySelector('[data-gu-marker=pinned-post]')
+	: null;
+
+const pinnedPostCheckBox: Element | null = !isServer
+	? window.document.querySelector('input[name=pinned-post-checkbox]')
+	: null;
+
+/**
+ * hide show more button on pinned post
+ */
+function hideShowMore() {
+	const pinnedPostButton = document.querySelector<HTMLElement>(
+		'#pinned-post-button',
+	);
+	if (pinnedPostButton) pinnedPostButton.style.display = 'none';
+}
+
+export const PinnedPostLiveness = ({}) => {
```

Comment:
Two things:

1) You can delete the empty object and just have `()`. I don't know why you should do this but some linters require this so my brain has been conditioned to call it out
2) How about `EnhancePinnedPost`? Or something along those lines? Just because 'Liveness' in DCR's context signifies polling for updates but this island isn't really that